src/routes/docs/tooling/command-line/teams/+page.markdoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: article
+title: Deploy teams with CLI
+description: Efficiently deploy your Appwrite teams using the Command-Line Tool (CLI). 
+---
+
+{% info title="Before proceeding" %}
+Ensure you [**install**](/docs/tooling/command-line/installation#Installation) the CLI, [**login**](/docs/tooling/command-line/installation#get-started) to your Appwrite account, and [**initialize**](/docs/tooling/command-line/installation#Initialization) your Appwrite project.
+{% /info %}
+
+The Appwrite CLI can create teams to organize users. Teams can be used to grant access [permissions](https://appwrite.io/docs/products/auth/teams#permissions) to a group of users.
```

Comment:
> grant access permissions

This doesn't quite sound right 🧐

Reply:
`Teams can be used to configure permissions for a group of users.